tests/test_grpc.py

```diff
@@ -148,6 +153,23 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+@skip_launch_mapdl  # need to be able to start/stop an instance of MAPDL
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    
+    ip = '127.0.0.2'
+    mapdl = launch_mapdl(ip=ip)
+
+    output_file = mapdl._download_as_raw('.__tmp__.out')
+    output = output_file.decode()
+
+    ip_in_output = re.findall('Server listening on : (.*):', output)[0]
+    
+    mapdl.exit()
+
+    assert ip == ip_in_output
```

Comment:
```suggestion
    assert mapdl.ip == ip
```
I've removed `mapdl.exit` as mapdl will destruct when it gets out of scope.